recipes/wavesim/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "wavesim" %}
+{% set version = "0.1.0a2" %}
```

```suggestion
{% set version = "0.1.0a2" %}
```
This appears to be an alpha release, which we do not normally allow on conda-forge.  Are there plans to start making stable releases?